shared/js/smart-screen/scrollable.js

```diff
@@ -203,7 +203,6 @@
       // Determine initial focus (depends on reversed or not)
       var initNode = this.getItemFromNode(
                      this.getNode(this._isReversed ? this.length - 1 : 0));
-      this.silentFocus(initNode);
 
```

Comment:
This is for the case that when toggling edit mode on, we have to realign reference element and then trigger this line of calling silentFocus. Since this silentFocus call has the reason to exist. I will handle refocus back to the original focused item in the toggleEditMode method.